resources/scss/ceres/widgets/Header/_top-bar-widget.scss

```diff
@@ -216,3 +216,117 @@ html.ie { // stylelint-disable-line selector-no-qualifying-type
         min-height: 32px;
     }
 }
+
+// Currency & Country Select
+.cmp-country-settings,
+.cmp-currency-select {
+    position: fixed;
+    right: 0;
+    left: 0;
+    z-index: $zindex-7;
+    width: 100%;
+    background-color: $gray-900;
+
+    &.collapsing {
+        overflow: hidden;
+    }
+
+    .shipping-settings,
+    .language-settings {
+        ul {
+            @include media-breakpoint-down(sm) {
+                column-count: 2;
+            }
+        }
+    }
+
+    .shipping-settings {
+        @include media-breakpoint-down(md) {
+            margin-top: 1rem;
+        }
+    }
+
+    ul {
+        @include reset-list();
+        max-height: 50vh;
+        overflow-x: hidden;
+        overflow-y: auto;
+    }
+
+    li {
+        float: left;
+        width: 100%;
+
+        a {
+            display: inline-block;
+            padding: .4rem;
+            &:hover {
+                @include context-bg($primary);
+            }
+
+            & .is-disabled {
```

Comment:
Das Kompiliert zum dem Selektor `li .is-disabled a`. Dadurch greift der Style nicht. Ich würde folgenden Aufbau vorschlagen:

```
li {
  a {
    // ...
 }

  &.is-disabled {
    a {
      // ...
    }
  }

  &.active {
    a {
      // ...
    }
  }
}

Reply:
done